common/lib/xmodule/xmodule/js/src/video/08_video_speed_control.js

```diff
@@ -109,6 +109,7 @@ function (Iterator) {
             speedsContainer.html(speedsList.join(''));
             this.speedLinks = new Iterator(speedsContainer.find('.speed-option'));
             this.state.el.find('.secondary-controls').prepend(this.el);
+            this.setActiveSpeed("1.0"); // default state on initial render
```

Comment:
@muzaffaryousaf This is set in the initial render of the page. This sets a default of 1.0x speed in case one isn't set. I tested this without issues; are you having trouble changing speeds?